src/__tests__/index.tsx

```diff
@@ -6,7 +6,7 @@ import {bundleMDX} from '..'
 import {getMDXComponent} from '../client'
 
 // compiling and bundling is slow...
-jest.setTimeout(20000)
+//jest.setTimeout(20000)
```

Comment:
The tests complete fast enough now that this isn't an issue anymore!

I left it commented out as I'm not sure if we want to keep it for developers on less able machines?